docs/tokenomics/nns/proposal-requirements.md

```diff
@@ -0,0 +1,206 @@
+
+* **ClearProvisionalWhitelist (#NetworkEconomics)**: clears the provisional whitelist, which allows the listed principals to create canisters with cycles. The mechanism is only needed for bootstrap and testing and must be deactivated afterward.
+
+* **RemoveNodeFromSubnet (#SubnetManagement)**: remove a node from a subnet. It then becomes available for reassignment. The execution of this proposal changes an existing subnet record to remove a node. From the perspective of the NNS, this update is a simple update of the subnet record in the registry.
+
+* **SetAuthorizedSubnetworks (#Governance)**: informs the cycles minting canister that a certain principal is authorized to use certain subnetworks (from a list). Can also be used to set the “default” list of subnetworks that principals without special authorization are allowed to use.
+
+* **SetFirewallConfig (#SubnetManagement)**: change the Firewall configuration in the registry (configures which boundary nodes subnet blockchain replicas will communicate with).
+
+* **UpdateNodeOperatorConfig (#NodeAdmin)**: change a node operator’s allowance in the registry.
+
+* **StopOrStartNNSCanister (#NetworkCanisterManagement)**: stop or start an NNS canister.
+
+## ICP tokens
+ICP are native utility tokens that play three key roles in the network:
+
+* **Facilitating network governance**: ICP tokens can be locked to create neurons that participate in network governance by voting, through which they can earn economic rewards.
+
+* **Production of cycles for compute**: ICP provides a source store of value that can be converted into “cycles,” which power computation in the role of fuel that is burned when it is used. The NNS converts ICP to cycles at a variable rate, so chosen to ensure users of the network can always create new cycles at approximately constant cost concerning IMF SDR, such that the cost of acquiring fuel is predictable.
+
+* **Rewarding participants**: the network provides rewards to incentivize those playing important roles that enable the network to function, including a) the provision of “voting rewards” to those participating in governance, and b) the provision of “node provider rewards” to those operating the node machines that are hosting the network.
+
+## Ledger
+The ICP ledger is hosted within the NNS and records all balances of ICP in the manner of a spreadsheet. Each row is called an “account,” which has two fields (i.e., there are two “columns”):
+
+* **Account identifier (bytes)**: a unique value that is derived from the identity of the “principal” that “controls” the account. Currently, the principal must either be (i) the owner of a public key pair, or (ii) a canister smart contract that is part of the NNS. Account identifiers are derived by hashing the concatenation of a domain separator, the principal ID, and the subaccount (or zeros if no subaccount is given).
```

Comment:
1. The controller (or owner) of the account is any principal. A canister that is not part of the NNS is still a valid owner.
2. Is there a reason why we do the distinction between principals associated with a key pair and the principal of a canister? Both are principals from the Ledger point of view.
3. Do we explain what the subaccount is? It is used but not introduced before.

Reply:
I think it is noted that a canister can vote as well. I think it might be too much to introduce subaccount to start